WhatsApp/app/src/main/java/com/example/whatsapp/ui/fragments/home/calls/CallsFragment.kt

```diff
@@ -106,6 +106,11 @@ class CallsFragment : BaseFragment<FragmentCallsBinding>(), ActionMode.Callback,
 
     }
 
+    override fun onPause() {
+        tracker?.clearSelection()
+        super.onPause()
```

Comment:
I don't think this way will work as `onPause` is also called when you move the app into the background and we would not want to clear our selection in that instance

Reply:
This should fix it c34973a16b8752e1364bbf7904a2549e90ab874b